homeassistant/components/drop_connect/sensor.py

```diff
@@ -0,0 +1,280 @@
+        key=CAPACITY_REMAINING,
+        translation_key=CAPACITY_REMAINING,
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.drop_api.capacity_remaining(),
+        state_class=SensorStateClass.TOTAL,
+    ),
+    DROPSensorEntityDescription(
+        key=CURRENT_SYSTEM_PRESSURE,
+        translation_key=CURRENT_SYSTEM_PRESSURE,
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PSI,
+        suggested_display_precision=1,
+        value_fn=lambda device: device.drop_api.current_system_pressure(),
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DROPSensorEntityDescription(
+        key=HIGH_SYSTEM_PRESSURE,
+        translation_key=HIGH_SYSTEM_PRESSURE,
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PSI,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.drop_api.high_system_pressure(),
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DROPSensorEntityDescription(
+        key=LOW_SYSTEM_PRESSURE,
+        translation_key=LOW_SYSTEM_PRESSURE,
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PSI,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.drop_api.low_system_pressure(),
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DROPSensorEntityDescription(
+        key=BATTERY,
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement=PERCENTAGE,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.drop_api.battery(),
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DROPSensorEntityDescription(
+        key=TEMPERATURE,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        suggested_display_precision=1,
+        value_fn=lambda device: device.drop_api.temperature(),
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DROPSensorEntityDescription(
+        key=INLET_TDS,
+        translation_key=INLET_TDS,
+        icon=TDS_ICON,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.drop_api.inlet_tds(),
+    ),
+    DROPSensorEntityDescription(
+        key=OUTLET_TDS,
+        translation_key=OUTLET_TDS,
+        icon=TDS_ICON,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.drop_api.outlet_tds(),
+    ),
+    DROPSensorEntityDescription(
+        key=CARTRIDGE_1_LIFE,
+        translation_key=CARTRIDGE_1_LIFE,
```

Comment:
Cartridge life sounds a bit diagnostic too. Please check.

Reply:
This is the percentage of life remaining in the user-replaceable filter cartridges.  I'd agree that it's diagnostic in nature.  The entity category was added in 3766113934397bf3d508ec3ddb37f27c6f2d5868.